drivers/windows/overlay/ov_endpoint_windows.go

```diff
@@ -155,7 +155,7 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 
 	hnsEndpoint.Policies = append(hnsEndpoint.Policies, paPolicy)
 
-	if osversion.Get().Build > 16236 {
+	if osversion.Get().Build >= osversion.RS3 {
```

Comment:
@pradipd @nwoodmsft  please double-check if this is OK (see my top comment); I suspect `16236` referred to `RS2`, which is not included in the constants (perhaps it should, even if it wasn't a Windows Server build?)

There's some more like these below

Reply:
I think ^ should be safe. RS3 EOL is right around the corner too, so even Build > osversion.RS3 should be okay in terms of docker engine support.

Reply:
Yes. that is correct.  Thanks!